Doc/library/time.rst

```diff
@@ -388,6 +388,22 @@ Functions
       by a signal, except if the signal handler raises an exception (see
       :pep:`475` for the rationale).
 
+.. function:: sleep_until(secs)
+
+   Like :func:`sleep`, but sleep until the specified time of the system clock (as
+   returned by :func:`time`). This can be used, for example, to schedule events
+   at a specific timestamp obtained from
+   :meth:`datetime.timestamp <datetime.datetime.timestamp>`.
+
+   See the notes in :func:`sleep` on the behavior when interrupted and on accuracy.
+   Additional potential sources of inaccuracies include:
+
+   * Because this function uses the system clock as a reference, this means the
+     reference clock is adjustable and may jump backwards.
+   * On Unix, if ``clock_nanosleep()`` is not available, the absolute timeout
+     is emulated using ``nanosleep()`` or ``select()``.
```

Comment:
If you consider that it's important to document the implementation, I suggest to do it in a separated section, since it's unclear to me what's the relationship between the chosen implementation and inaccuracies. If you care about the theorical clock resolution, you should document it, it's not obviously to users what is the resolution of these functions.